target/linux/ath79/image/generic.mk

```diff
@@ -1015,12 +1015,14 @@ define Device/dlink_dir-825-b1
   DEVICE_VENDOR := D-Link
   DEVICE_MODEL := DIR-825
   DEVICE_VARIANT := B1
-  IMAGE_SIZE := 6208k
-  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
-	check-size | append-metadata
   DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport \
 	kmod-leds-reset kmod-owl-loader kmod-switch-rtl8366s
-  SUPPORTED_DEVICES += dir-825-b1
```

Comment:
Why is the upgrade from 19.07 not working any more? 
The old image should be only in `fwconcat0`.

Reply:
@hauke On 19.07 firmware partition is only 6208k (fwconcat0) and now on master it is 7808k(fwconcat0+fwconcat1).
While it would still upgrade fine if sysupgrade image size does not go over 6208k, it is just a matter of time when it will. So to prevent upgrade failures it should be better to disable upgrade support by default. If user really knows that he is doing, and checks that  upgrade image fits to 6208k , he can still force update.
Same logic is used for recently ported TrendNet TEW-673GRU (12cee869890853716ff1ee2dbd0a89c87a0ee544), which uses same partition table.
Also it would be quite a big version jump from 19.07 to 23.xx, in which these changes would be applied. Possible jump, but nevertheless not recommended.
Let me know what you think.